src/index.js

```diff
@@ -11,6 +11,10 @@ import {
   Dispatcher
 } from './internals'
 
+// `isServer` will be `true` in node & jest, & `false` in
+// the browser where the bundler shims the `process` object
+const isServer = !process.browser
```

Comment:
Let's maybe make this more explicit? There's no need, technically, for us to use `process.browser` since we can just go on `setImmediate` itself :)

```suggestion
// In the presence of setImmediate, i.e. on Node, we'll enable the
// yielding behavior that gives the event loop a chance to continue
// running when the prepasses would otherwise take too long
const SHOULD_YIELD = typeof setImmediate === 'function';
```

Reply:
@kitten Yup, I agree your way is more simple.